crates/fluvio-connector/test-config.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: my-test-connector
+type: test-connector
+topic: foobar
+direction: source
+args:
```

Comment:
This is out of date, I think this file should only exist if its being used by a test, in order to make sure its valid.

Reply:
Hmm. True. @sehz should I add a simple test to ci?

Reply:
Definitely!

Reply:
I think that can as separate PR

Reply:
Well, I added it. It's nothing fancy. Just a simple test to load the config. Doesn't run it against the sc but that's good enough for now.